csrankings-n.csv

```diff
@@ -450,6 +450,7 @@ Nikitas M. Sgouros,University of Piraeus,https://www.ds.unipi.gr/en/faculty/sgou
 Niklas Broberg,Chalmers/GU,https://www.chalmers.se/en/staff/Pages/d00nibro.aspx,NOSCHOLARPAGE
 Niklas Elmqvist,University of Maryland - College Park,https://sites.umiacs.umd.edu/elm,LoQXe24AAAAJ
 Niko Sunderhauf,Queensland University of Technology,https://research.qut.edu.au/qcr/Our%20people/niko-suenderhauf,WnKjfFEAAAAJ
+Niko Sünderhauf,Queensland University of Technology,https://research.qut.edu.au/qcr/Our%20people/niko-suenderhauf,WnKjfFEAAAAJ

```

Comment:
It looks like your name without the umlaut is no longer listed in DBLP as an alias; please go ahead and remove that entry and I will merge. Thanks.